Umbraco-Cloud/Upgrades/Migrating-from-9-to-10/index.md

```diff
@@ -0,0 +1,130 @@
+---
+versionFrom: 9.0.0
+---
+
+# Upgrading an Umbraco 9 Cloud project to Umbraco 10
+
+This article will provide steps on how to upgrade an Umbraco 9 Cloud project to Umbraco 10.
+
+The upgrade path between Umbraco 9 and Umbraco 10 can be done directly by updating your project using NuGet. You will need to ensure the packages you are using are available in Umbraco 10.
+
+## Prerequisites
+
+* An Umbraco 9 Cloud project running **the latest version of Umbraco 9** with **at least 2 environments**.
+
+* A backup of your Umbraco 9 project database.
+
+:::note
+We strongly recommend having at least 2 environments on your project. If something fails during the upgrade, the Development environment can be removed and added again to start over with the upgrade process.
+:::
+
+## Step 1: Content Migration
+
+* Create a backup of the database from your project, see the [Database backups](https://our.umbraco.com/documentation/umbraco-cloud/Databases/Backups/) article or clone down the project and take a backup of the local Database (Make sure to restore the content from your cloud environment).
+
+* On the Cloud portal, go to the project that you wish to upgrade and navigate to **Settings** -> **Advanced**. Scroll down to **Runtime Settings** section and **Enable .NET 6 for your Umbraco 9 install** for each environment of your cloud project.
+
+    ![Runtime Settings](images/Runtime-Settings.png)
+
+* Clone down the **Development** environment from the Umbraco Cloud project, build and run the project locally and make sure to log in to the backoffice.
```

Comment:
Should there also be a step for restoring content? I guess you would wanna test the upgrade against that, right?

```suggestion
* Clone down the **Development** environment from the Umbraco Cloud project.
* Build and run the project locally.
* Make sure to log in to the backoffice.
* Restore content from your Cloud enviroment.
```